app/controllers/accounts_controller.rb

```diff
@@ -26,10 +27,11 @@ def show
       end
 
       format.json do
-        render json: @account,
-               serializer: ActivityPub::ActorSerializer,
-               adapter: ActivityPub::Adapter,
-               content_type: 'application/activity+json'
+        skip_session!
+
+        render_cached_json(['activitypub', 'actor', @account.cache_key], content_type: 'application/activity+json') do
+          ActiveModelSerializers::SerializableResource.new(@account, serializer: ActivityPub::ActorSerializer, adapter: ActivityPub::Adapter)
+        end
```

Comment:
The rest seems fine, but isn't this going to cause Mastodon to serve stale account info (display name, avatar, header image, etc.)?

Reply:
@ThibG: No. `#cache_key` method returns a combination of model name, ID and updated_at

Reply:
Alright, it's fine with me, then!